app/views/user/show.html.erb

```diff
@@ -1 +1,16 @@
-<h1>List of users for the given id</h1>
\ No newline at end of file
+<section id='user-container'>
+<img src="<%= @user.photo %>" alt="image">
+<h1><%= @user.name%></h1>
+<p>Number of post: <%= @user.posts.length%></p>
+</section>
+<p id='user-bio'><%= @user.bio %></>
+<div class='post-container'>
+<% @user.recent_posts.each_with_index do |post,index| %>
+<li>
+<p>Post #<%= index + 1%></p>
+ <h3><%= link_to post.title, controller: "posts", action: "show", user_id: @user.id, post_id: post.id %></h3>
+<p>Comments: <%+ post.comments.length %></p>
+<p>Likes: <%+ post.likes.length %></p>
+</li>
+<% end %>
+</div>
```

Comment:
- [ ] Kindly make your user's show page look like that of the wireframe provided. On the show page of the user, there should be a _**See all posts**_ button. Kindly implement this change.

### Wireframe

![image](https://user-images.githubusercontent.com/77106962/224071789-739abd80-3d6c-44d2-b4c7-50049b9b5d54.png)